server/svix-server/src/v1/endpoints/message.rs

```diff
@@ -347,10 +386,20 @@ async fn expunge_message_content(
 
 pub fn router() -> ApiRouter<AppState> {
     let tag = openapi_tag("Message");
+
+    fn document_create_message(op: TransformOperation) -> TransformOperation {
```

Comment:
This PR can go in, though there's one question to answer before we can (which I think I asked already? If so, sorry for repeating myself!): how do the auto-generated libs deal with this new multi-status point API? E.g. what does the generated Rust lib does? TypeScript? Python? Java?

Reply:
Hmm, it's actually not great 😞 
I tried regenerating after adding a response code `200` response to this operation, with no response body. I checked two languages:
1. In Go, it removed the automatic deserialization from the response, so the generated method just returns the http response to you and it's up to you to deserialize the correct response.
2. In Rust it's even worse because it seems if there's a response with code 200 it prefers that and disregards other successful response codes. In this case it changed it from `Result<MessageOut, SomeErrorType>` to `Result<(), SomeErrorType>`.

Reply:
I scoured the OpenAPI generator issue tracker a little bit and it seems that support for this varies language by language:
* https://github.com/OpenAPITools/openapi-generator/issues/7714 - Ruby is wrong
* https://github.com/OpenAPITools/openapi-generator/issues/7426 - Python might be correct, or at least the issue is marked resolved
* In the mean time I also checked Java locally, which is also wrong.

So yeah, we might not want to add another response code, rather add a different response type maybe? 😕